Code/6-wire-version/src/main.cpp

```diff
@@ -454,7 +604,6 @@ void loadWifi()
   enableAp = doc["enableAp"];
   apSsid = doc["apSsid"].as<String>();
   apPwd = doc["apPwd"].as<String>();
-  Serial.println("loading pwd:   " + apPwd);
```

Comment:
debugging left-over. A call to WiFi.disconnect() made this variable to be garbage. Maybe some memory got written over, or the pointer changed somehow.